tls/s2n_handshake_io.c

```diff
@@ -1077,6 +1078,57 @@ static int s2n_handle_retry_state(struct s2n_connection *conn)
     return 0;
 }
 
+static int s2n_write_keylog(struct s2n_connection *conn)
```

Comment:
If we don’t care about failure, should we make this  `void`?

Same on the callback.

Reply:
Using `int` allows to use the `GUARD` macro. ACK on the callback, will change.